source/slang/slang-serialize-ir.cpp

```diff
@@ -195,6 +195,14 @@ Result IRSerialWriter::write(IRModule* module, SerialSourceLocWriter* sourceLocW
                 switch (srcInst->getOp())
                 {
                     // Special handling for the ir const derived types
+                    case kIROp_BlobLit:
+                    {
+			// Blobs are serialized into string table like strings
```

Comment:
The indentation here is off, because it uses tab characters.
We use space characters for indentations.